src/main/java/org/isf/medicalstockward/gui/WardPharmacyRectify.java

```diff
@@ -411,25 +411,21 @@ public void actionPerformed(ActionEvent e) {
 							if (ok != JOptionPane.OK_OPTION) return;
 						}
 						
-						boolean result;
 						try {
 							
 							movStockInsertingManager.storeLot(selectedLot.getCode(), selectedLot, med);
-							result = movWardBrowserManager.newMovementWard(new MovementWard(
-									selectedWard, 
+							movWardBrowserManager.newMovementWard(new MovementWard(
+									wardSelected, 
```

Comment:
Sorry, my mistake in resolving conflict from the online GitHub editor... could you please change to `selectedWard`? 
Thanks!

Reply:
nevermind, I do it by myself after the merge because checks are failing anyway. Thanks!